### DISCUSSION TITLE
Updating GitHub Pages causes it to 404

### DISCUSSION THREAD

**Discussion by NeedsMoreGrey** [community/community#137600] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm using Mkdocs Material combined with GitHub to host a very small site for my friends on a custom domain pointed at my GitHub Pages.

The site was working. 

I pushed an update to it, only changing some of the text on the website. Now the site 404s, but only on the custom domain.

The GitHub Pages version works and I have not modified any of the DNS settings.

Thoroughly confused.

**Comment by ghost** [comment]
Try republishing it if you are still having this issue.
